crosschain/three-chains-six-contracts/src/main/java/tech/pegasys/samples/crosschain/threechainssixcontracts/ThreeChainsSixContracts.java

```diff
@@ -61,11 +61,14 @@
     private static final BigInteger SC0_SIDECHAIN_ID = BigInteger.valueOf(11);
     private static final String SC0_URI = "http://127.0.0.1:8110/";
```

Comment:
Why is the port for SCO an embedded constant? If SC0 is always for Crosschain coordination, is it better to name it COORDINATION_SIDECHAIN_ID?

Reply:
This sounds like a good idea. I think we should generalise this set-up in a separate file, that all of the samples inherit. In this way, we won't need to copy this code to each sample.